src/storage-controller/Cargo.toml

```diff
@@ -33,8 +34,16 @@ proptest = { version = "1.0.0", default-features = false, features = ["std"] }
 prost = { version = "0.11.3", features = ["no-recursion-limit"] }
 serde = { version = "1.0.152", features = ["derive"] }
 serde_json = { version = "1.0.89" }
-timely = { version = "0.12.0", default-features = false, features = ["bincode"] }
-tokio = { version = "1.24.2", features = ["fs", "rt", "sync", "test-util", "time"] }
+timely = { version = "0.12.0", default-features = false, features = [
```

Comment:
I don't know why my autoformatter started doing this. Will see if I can fix it on next push.